packages/router/src/util.ts

```diff
@@ -210,9 +210,9 @@ const replaceParams = (path: string, args: Record<string, unknown> = {}) => {
         const paramSpec = part.substr(1, part.length - 2)
         const paramName = paramSpec.split(':')[0]
         const arg = args[paramName]
-        if (arg) {
+        if (Object.prototype.hasOwnProperty.call(args, paramName)) {
           delete args[paramName]
-          return arg
+          return '' + arg
```

Comment:
Interesting workaround. Is this a "known" way to resolve this issue were you want to return the value and not a boolean? 'Cause I need to remember it!

Reply:
It's to convert `arg` to a string. I've added a comment in the code to clarify this.